docs/source/distributions/configuration.md

```diff
@@ -385,6 +385,121 @@ And must respond with:
 
 If no access attributes are returned, the token is used as a namespace.
 
+### Access control
+
+When authentication is enabled, access to resources is controlled
+through the `access_policy` attribute of the auth config section under
+server. The value for this is a list of access rules.
```

Comment:
I would probably note the available list of available actions (create, read, update, delete) just for completeness.

Reply:
update now added to the valid list of actions.